lib/src/main/java/fi/epicbot/toster/context/ActionContext.kt

```diff
@@ -14,6 +14,11 @@ class ActionContext {
         delay(delayMillis)
     }
 
+    fun longClick(x: Int, y: Int, clickDelayMs: Long = 200L, delayMillis: Long = 0) {
```

Comment:
Let's add description there, at least, why we should set clickDelayMs

Reply:
done